pkg/quota/image/registry.go

```diff
@@ -11,19 +11,29 @@
 package image
 
 import (
+	"fmt"
+	"time"
+
 	"k8s.io/kubernetes/pkg/api/unversioned"
+	"k8s.io/kubernetes/pkg/client/cache"
 	"k8s.io/kubernetes/pkg/quota"
 	"k8s.io/kubernetes/pkg/quota/generic"
 
 	osclient "github.com/openshift/origin/pkg/client"
+	imageapi "github.com/openshift/origin/pkg/image/api"
+)
+
+const (
+	imageTTL time.Duration = time.Minute * 30
```

Comment:
None. We don't need to list periodically all images. Just those referenced by image streams in prijects were image quota or limit range is set.